.github/workflows/publish_release.yml

```
@@ -32,3 +32,40 @@ jobs:
             - name: Build and Push Release to DockerHub
               shell: bash
               run: ./dev-scripts/build_and_push_release.sh
+    draft-github-release:
+        needs: build-publish-docker-hub
+        permissions:
+            contents: write
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)